cmd/ui.go

```diff
@@ -88,12 +90,55 @@ func renderMultipleBars(isTTY, goBack bool, leftMax int, pbs []*pb.ProgressBar)
 		lineEnd = "\x1b[K\n" // erase till end of line
 	}
 
-	pbsCount := len(pbs)
-	result := make([]string, pbsCount+2)
+	var (
+		// Maximum length of each right side column except last,
+		// used to calculate the padding between columns.
+		maxRColumnLen = make([]int, 1)
```

Comment:
This seems a bit strange, but I guess it's going to be addressed in the next PR anyway...

Reply:
No, this mostly stays the same, and I agree that it's strange. :)

Initially I implemented a dumber algorithm that only worked for two columns, but this approach allows us to properly align N columns, by expanding this slice and returning more than two elements in the `right` slice returned by `progresFn`. Ultimately it's much cleaner and flexible, at the expense of slight weirdness right now with only two columns.

Let me know if I should comment this in the code.